pymc3/step_methods/hmc/hmc.py

```diff
@@ -50,18 +55,40 @@ def __init__(self, vars=None, path_length=2., step_rand=unif, **kwargs):
             An object that represents the Hamiltonian with methods `velocity`,
             `energy`, and `random` methods. It can be specified instead
             of the scaling matrix.
+        target_accept : float (0,1), default .8
+            Try to find a step size such that the average acceptance
+            probability across the trajectories are close to target_accept.
+            Higher values for target_accept lead to smaller step sizes.
+        gamma : float, default .05
+        k : float (.5,1) default .75
```

Comment:
.5,1?

Reply:
I just copied that from nuts, so if we change it here, we should also change it there.
k has to be in the open interval (0.5, 1).

Reply:
I see, isn't `[0.5, 1]` the more common syntax?

Reply:
That would imply a closed interval, including 0.5 and 1. I don't think that's allowed. I can just put it in words in the description, that would be leas confusing I think.

Reply:
I think common syntax might be `]0.5, 1[`, which makes me twitch, but at least makes it clear we're not talking about a tuple.  also fine with using words!

Reply:
Please dont use `]0.5, 1[`... word is good

Reply:
Words it is.
I think the ]0.5, 1[ syntax is more common in the US, and (0.5, 1) in europe.